docs/src/componentDocs/Header/markdown/HeaderExamples.mdx

```diff
@@ -1 +1,189 @@
-### Header Examples
+import Box from '@mui/material/Box';
+import headerImageExample from '../images/headerImage.png';
+import headerBackgroundExample from '../images/headerBackgroundImage.png';
+import headerActionsExample from '../images/headerActionsImage.png';
+import headerSearchExample from '../images/headerSearchImage.png';
+import headerCollapsedExample from '../images/headerCollapsedImage.png';
+import headerExpandedExample from '../images/headerExpandedImage.png';
+
+## Header
+
+The `<Header>` component is used at the top of a page to display information. It displays a title and can optionally show a subtitle, a background image, a navigation button, and multiple action buttons. The header can also be configured to expand or collapse as needed. For expand/collapse behavior tied to the screen scroll position, you should use the [`<CollapsibleHeaderLayout>`](/components/collapsible-header-layout/examples) component.
```

Comment:
```suggestion
A basic `Header` component is used at the top of the screen and can show a title and (optional) subtitle. A third line of text can also be configured using the `info` prop, but it will only be visible when the `Header` is expanded (see below). 
```